widgets/Alert.php

```diff
@@ -51,21 +51,18 @@ class Alert extends \yii\bootstrap4\Widget
     public function run()
     {
         $session = Yii::$app->session;
-        $flashes = $session->getAllFlashes();
         $appendClass = isset($this->options['class']) ? ' ' . $this->options['class'] : '';
 
-        foreach ($flashes as $type => $flash) {
-            if (!isset($this->alertTypes[$type])) {
-                continue;
-            }
+        foreach (array_keys($this->alertTypes) as $type) {
+            $flash = $session->getFlash($type);
 
             foreach ((array) $flash as $i => $message) {
                 echo \yii\bootstrap4\Alert::widget([
                     'body' => $message,
                     'closeButton' => $this->closeButton,
                     'options' => array_merge($this->options, [
-                        'id' => $this->getId() . '-' . $type . '-' . $i,
-                        'class' => $this->alertTypes[$type] . $appendClass,
+                        'id' => $this->getId().'-'.$type.'-'.$i,
```

Comment:
Please keep the spaces, this is not Symfony 😀

Reply:
OK I'll remember that in future :smile: Do I have to push again in the branch I sent the pull request with the added spaces? (as said before, I'm new to GitHub and all the workflows and I don't want to mess up everything, sorry for that newbie questions :laughing: )

Reply:
Just push it to the same branch ;)